content/en/idempotence.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Idempotence
+status: Feedback Appreciated
+category: concept
```

Comment:
Looks like this should be a property in which case, there is no breakdown in "what it is," "problem it addresses," and "how it helps." See [style guide](https://glossary.cncf.io/style-guide/#three-subheadings).